web/src/features/hfiCalculator/components/DailyViewTable.tsx

```diff
@@ -355,90 +355,81 @@ export const DailyViewTable = (props: Props): JSX.Element => {
+                          ] as KeyTooltipDecimalPlacesTuple,
+                          [
+                            'precipitation' as keyof StationDaily,
                             'Precipitation cannot be null. Impacts DC, BUI, ROS, HFI, FIG, Prep calculations.'
-                          }
-                        />
-                        <RequiredDataCell
-                          classNameForRow={classNameForRow}
-                          dailyKey={'ffmc'}
-                          daily={daily}
-                          errorToolTipText={
-                            'FFMC cannot be null. Impacts ROS, HFI, Fire Type, FIG, Prep calculations.'
-                          }
-                        />
-                        <RequiredDataCell
-                          classNameForRow={classNameForRow}
-                          dailyKey={'dmc'}
-                          daily={daily}
-                          errorToolTipText={
-                            'DMC cannot be null. Impacts ROS, HFI, Fire Type, FIG, Prep calculations.'
-                          }
-                        />
-                        <RequiredDataCell
-                          classNameForRow={classNameForRow}
-                          dailyKey={'dc'}
-                          daily={daily}
-                          errorToolTipText={
-                            'DC cannot be null. Impacts ROS, HFI, Fire Type, FIG, Prep calculations.'
-                          }
-                        />
-                        <RequiredDataCell
-                          classNameForRow={classNameForRow}
-                          dailyKey={'isi'}
-                          daily={daily}
-                          errorToolTipText={
-                            'ISI cannot be null. Impacts ROS, HFI, Fire Type, FIG, Prep calculations.'
-                          }
-                        />
-                        <RequiredDataCell
-                          classNameForRow={classNameForRow}
-                          dailyKey={'bui'}
-                          daily={daily}
-                          errorToolTipText={
-                            'BUI cannot be null. Impacts ROS, HFI, Fire Type, FIG, Prep calculations.'
-                          }
-                        />
-                        <RequiredDataCell
-                          classNameForRow={classNameForRow}
-                          dailyKey={'fwi'}
-                          daily={daily}
-                          errorToolTipText={
-                            'FWI cannot be null. Impacts ROS, HFI, Fire Type, FIG, Prep calculations.'
-                          }
-                        />
+                          ] as KeyTooltipDecimalPlacesTuple,
+                          [
+                            'ffmc' as keyof StationDaily,
+                            'FFMC cannot be null. Impacts ROS, HFI, Fire Type, FIG, Prep calculations.',
+                            0
```

Comment:
Seems like none of these parameters differ, can we just use a constant?

Reply:
The params do differ somewhat.

Reply:
Does `0` differ?

Reply:
I think you might be looking at an older version - try refreshing the page. I've removed the 0s for now (they might return later if/when we decide to remove decimal points from some of our values)

Reply:
Sorry, when you said none of the params differ, I thought you were talking about the contents of error message tooltips.

Reply:
Nope, my bad, I should have specified which parameter. LGTM.